src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -0,0 +1,627 @@
+package org.jitsi.impl.neomedia.rtp.remotebitrateestimator;
+
+import org.ice4j.util.RateStatistics;
+import org.jitsi.impl.neomedia.codec.audio.silk.K2aFLP;
+import org.jitsi.impl.neomedia.transform.PacketTransformer;
+import org.jitsi.service.neomedia.RawPacket;
+import org.jitsi.service.neomedia.rtp.RemoteBitrateEstimator;
+import org.jitsi.util.Logger;
+
+import javax.media.rtp.RTPHeader;
+import java.util.*;
+
+/**
+ * webrtc.org abs_send_time implementation as of June 26, 2017.
+ * @author Julian Chukwu
+ */
+public class RemoteBitrateEstimatorAbsSendTime
+    implements RemoteBitrateEstimator
+{
+    private static final Logger logger = Logger
+            .getLogger(RemoteBitrateEstimatorAbsSendTime.class);
+
+    private enum probeResult
+    {
+        kBitrateUpdated(0),
+        kNoUpdate(1);
+        int value;
+        probeResult(int x)
+        {
+            this.value = x;
+        };
+    }
+
+    private final Object critSect = new Object();
+
+    public final static int kTimestampGroupLengthMs = 5;
+    public final static int kAbsSendTimeFraction = 18;
+    public final static int kAbsSendTimeInterArrivalUpshift = 8;
+    public final static int kInterArrivalShift
+            =  kAbsSendTimeFraction + kAbsSendTimeInterArrivalUpshift;
+    public final static int kInitialProbingIntervalMs = 2000;
+    public final static int kMinClusterSize = 4;
+    public final static int kMaxProbePackets = 15;
+    public final static int kExpectedNumberOfProbes = 3;
```

Comment:
Are there going to be used from outside this class? If not, they should be private.